master/usage/configuration/node.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Configuring a node
+---
+
+Each Calico node has key routing configuration that may be specified in a number of
+different ways.  In particular, this includes the IPv4 and IPv6 addresses and 
+subnets used for routing, and the BGP AS Number (which generally only needs to 
+be changed if you are explicitly configuring BGP peerings with your Calico nodes).
+
+To configure the routing parameters on your node, you have the options of:
+-  Set appropriate flags on the `calicoctl node run` command
+-  Set appropriate environments for the `calico/node` container
+-  Configure the node resource using `calicoctl`
+
+> Note that for nodes that were configured prior to v2.1, the subnet information 
+> configured on those nodes will be set to /32 for IPv4 or /128 for IPv6.  If you are
+> planning on using the `cross-subnet IP-in-IP` feature then you must update your node
+> configuration to have the correct subnet information for each node.
+
+## Configuring the IP address and subnet
+
+**Configure the IP and subnet through runtime args/environments**
+
+The IPv4 address may be explicitly specified using the `--ip` option on 
+`calicoctl node run` or the `IP` environment if you are starting the container
+directly.  For IPv6, use the equivalent `--ip6` option and `IP6` environment.
+
+If you are currently specifying the IP address, you will need to update the
```

Comment:
I don't like this wording now that this is in a stand alone page.  Maybe something like
> To specify the IP address, the parameter must include the IP and network in CIDR form (e.g. `10.0.2.10/24`).

I guess the note above about upgrading to v2.1 could be moved into this section but I could go either way on that.